agents/labjack/labjack_agent.py

```diff
@@ -206,7 +206,11 @@ def __init__(self, agent, ip_address, active_channels, function_file,
                                  record=True,
                                  agg_params=agg_params_downsampled,
                                  buffer_time=1)
-
+        self.agent.register_feed('registers',
+                                 record = True,
+                                 agg_params = {'frame_length':10*60},
+                                 buffer_time = 0.)
```

Comment:
All other feeds in the Agent use a 1 second buffer. Any reason not to here?

Reply:
No, changed to 1 second.